ts/features/fci/screens/valid/FciDataSharingScreen.tsx

```diff
@@ -81,7 +86,15 @@ const FciDataSharingScreen = (): React.ReactElement => {
       <H4 weight="Regular" style={styles.paddingTextLarge} color={"bluegrey"}>
         {I18n.t("features.fci.shareDataScreen.alertText")}
         <View style={styles.paddingText} />
-        <Link>{I18n.t("features.fci.shareDataScreen.alertLink")}</Link>
+        <Link
+          onPress={() =>
+            navigation.navigate(ROUTES.PROFILE_NAVIGATOR, {
```

Comment:
It seems we've not properly handled the navigation stack:

https://user-images.githubusercontent.com/16268789/226974712-c923c970-40b0-44d7-abe6-9c6770f4a60f.mp4

Reply:
Tapping on "Modify your e-mail" button, give us this error.

Reply:
This depends on `RemindEmailValidationOverlay`.  There's yet a temp workaround related email / isOnboardingCompleted screens. Will be removed in https://pagopa.atlassian.net/browse/IAI-139
I tried to fix it with temp wa here [8fcccfe](https://github.com/pagopa/io-app/pull/4470/commits/8fcccfe13b9d7b656cf0858a5681c8e6a243f419)

Reply:
Okay, Got it! Good to know 💪